sigh/lib/assets/resign.sh

```diff
@@ -193,8 +195,13 @@ while [ "$1" != "" ]; do
             BUNDLE_VERSION="$1"
             ;;
         --use-app-entitlements )
+            shift
```

Comment:
`--use-app-entitlements` doesn't take a parameter, so there shouldn't be a `shift` here. This breaks the `use-app-entitlements` option when using resign from fastlane :(